### PR TITLE
[it] Probabilities and Statistics

### PR DESCRIPTION
**Pull request by Fodark**

### CODE REVIEW

**Comment by shervinea** [comment]
Thank you @Fodark for all your hard work! Excited to see the Italian version coming to life. Please feel free to invite anyone you know who could review your work.

File: it/cs-229-probability.md [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+
+**47. Random sample ― A random sample is a collection of n random variables X1,...,Xn that are independent and identically distributed with X.**
+
+&#10230; Campione casuale ― Un campione casuale è un gruppo di n variabili aleatorie X1,...,Xn distribuite in modo indipendente e indentico con X.
+
+<br>
+
+**48. Estimator ― An estimator is a function of the data that is used to infer the value of an unknown parameter in a statistical model.**
+
+&#10230; Stimatore ― Uno stimatore è una funzione dei dati usata per dedurre il valore di un parametro sconosciuto in un  modello statistico.
```
Comment (aepiotti) [inline it/cs-229-probability.md]: &#10230; Stimatore ― Uno stimatore è una funzione dei dati usata per dedurre il valore di un parametro sconosciuto in un modello statistico.

Comment (aepiotti) [reply it/cs-229-probability.md]: double *space* character between "un" and "modello"
The only error/typo I found

**Comment by Fodark** [comment]
Thank you @aepiotti for your review!
Fixed with your suggestion and added myself to the contributors!

**Comment by shervinea** [comment]
Thank you both for your time and help, it is appreciated! Now moving forward in merging this PR.

**Pull request merged**